examples/CRISP/scripts/local_dev/run_aggregator.sh

```
@@ -1,19 +1,7 @@
-ENCLAVE_CONTRACT="0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512"
-REGISTRY_CONTRACT="0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0"
-FILTER_REGISTRY_CONTRACT="0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9"
-PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
-NODE_ADDRESS="0x8626a6940E2eb28930eFb4CeF49B2d1F2C9C1199"
-PASSWORD="We are the music makers and we are the dreamers of the dreams."
-QUIC_PORT=9204
-
-# Setup directories
-SCRIPT_DIR=/tmp/enclave-nodes
 DATA_DIR="$SCRIPT_DIR/enclave_data/aggregator"
 LOG_FILE="$DATA_DIR/aggregator.log"
```

Shellcheck (0.10.0)

[warning] LOG_FILE appears unused. Verify use (or export if used externally).

(SC2034)